CodePush.m

```diff
@@ -351,9 +386,7 @@ - (void)savePendingUpdate:(NSString *)packageHash
             [self savePendingUpdate:updatePackage[PackageHashKey]
                           isLoading:NO];
             
-            if (installMode == CodePushInstallModeImmediate) {
-                [self loadBundle];
-            } else if (installMode == CodePushInstallModeOnNextResume) {
+            if (installMode == CodePushInstallModeOnNextResume) {
```

Comment:
Was the fix for the immediate-applied update big simply removing this call to `loadBundle` and allowing the script-side to trigger the restart? Would this bug affect the Android plugin?